packages/cli-kit/src/public/node/template.ts

```diff
@@ -1,14 +1,17 @@
-import {glob, join, dirname, relative} from './path.js'
-import {mkdir, read, copy, chmod, isDirectory, write, hasExecutablePermissions} from './file.js'
-import {content, token, debug} from './output.js'
+import {glob, join, dirname, relative} from '../../path.js'
+import {mkdir, read, copy, chmod, isDirectory, write, hasExecutablePermissions} from '../../file.js'
+import {content, token, debug} from '../../output.js'
 import {Liquid} from 'liquidjs'
 
-// This line is necessary to register additional helpers.
-export function create(templateContent: string) {
-  return (data: object): Promise<string> => {
-    const engine = new Liquid()
-    return engine.render(engine.parse(templateContent), data)
-  }
+/**
+ * Renders a template using the Liquid template engine.
+ * @param templateContent - Template content.
+ * @param data - Data to feed the template engine.
+ * @returns Rendered template.
+ */
+export function renderTemplate(templateContent: string, data: object): Promise<string> {
```

Comment:
I'd extract this one into `public/common/liquid.ts` and rename it to `renderLiquidTemplate`.